vader.py

```diff
@@ -0,0 +1,63 @@
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+from nltk import tokenize
+import nltk
+import ssl
+
+try:
+    _create_unverified_https_context = ssl._create_unverified_context
+except AttributeError:
+    pass
+else:
+    ssl._create_default_https_context = _create_unverified_https_context
+nltk.download('vader_lexicon')
+
+
+
+#this function takes in a list of comments and outputs a corresponding list of sentiment scores ranging from -1 to 1
+def sentScore(comments):
+
+    #initalize an empty array for the scores
+    score = []
+
+    #initalize the model
+    sid = SentimentIntensityAnalyzer()
+
+    #stores the score for each comment in the score list
+    for comment in comments:
+        #a dictionary 
+        s =  sid.polarity_scores(comment)
+        
```

Comment:
The code structure is clear enough but Im not sure what 'compound' is or how the dictionary is structured.